docs/README.md

```diff
@@ -94,8 +94,8 @@ The following table displays the tested Kubernetes and Helm versions.
 | K8s with EKS           | 1.25<br/>1.26<br/>1.27<br/>1.28<br/>1.29 |
 | K8s with EKS (fargate) | 1.25<br/>1.26<br/>1.27<br/>1.28<br/>1.29 |
 | K8s with Kops          | 1.25<br/>1.26<br/>1.27<br/>1.28<br/>1.29 |
-| K8s with GKE           | 1.25<br/>1.26<br/>1.27<br/>1.28<br/>1.29 |
```

Comment:
Why do we drop gke 1.25? I see e2e tests passing for it

Reply:
It has reached EOL, we should drop the tests then: https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule-for-release-channels